loony_dev/tests/test_pr_review_task.py

```
@@ -193,5 +193,68 @@ def test_discover_skips_when_inline_before_marker(self) -> None:
+    """on_complete must always write a success marker, even when post_summary=False."""
+
+    def _make_pr(self, comment_ts: str = "2024-01-01T10:00:00Z") -> MagicMock:
+        pr = MagicMock()
+        pr.number = 1
+        pr.new_comments = [_comment(REVIEWER, "Review comment", comment_ts)]
+        return pr
+
+    def _make_repo(self) -> MagicMock:
+        return _mock_repo()
+
+    def _make_result(self, post_summary: bool, summary: str = "Some summary") -> "TaskResult":
```

Ruff (0.15.12)

[error] Undefined name `TaskResult`

(F821)